[2024-02-26 10:06:11,973] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: terceira_dag.tsk3 2024-02-26T10:05:58.530062+00:00 [queued]>
[2024-02-26 10:06:11,989] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: terceira_dag.tsk3 2024-02-26T10:05:58.530062+00:00 [queued]>
[2024-02-26 10:06:11,989] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2024-02-26 10:06:11,989] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2024-02-26 10:06:11,990] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2024-02-26 10:06:12,014] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): tsk3> on 2024-02-26T10:05:58.530062+00:00
[2024-02-26 10:06:12,019] {standard_task_runner.py:52} INFO - Started process 66 to run task
[2024-02-26 10:06:12,023] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'terceira_dag', 'tsk3', '2024-02-26T10:05:58.530062+00:00', '--job-id', '21', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/terceiradag.py', '--cfg-path', '/tmp/tmpr5qu4sse', '--error-file', '/tmp/tmp6hvx803q']
[2024-02-26 10:06:12,024] {standard_task_runner.py:77} INFO - Job 21: Subtask tsk3
[2024-02-26 10:06:12,083] {logging_mixin.py:109} INFO - Running <TaskInstance: terceira_dag.tsk3 2024-02-26T10:05:58.530062+00:00 [running]> on host aed67f908f56
[2024-02-26 10:06:12,162] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=terceira_dag
AIRFLOW_CTX_TASK_ID=tsk3
AIRFLOW_CTX_EXECUTION_DATE=2024-02-26T10:05:58.530062+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-26T10:05:58.530062+00:00
[2024-02-26 10:06:12,164] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2024-02-26 10:06:12,166] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 5']
[2024-02-26 10:06:12,183] {subprocess.py:74} INFO - Output:
[2024-02-26 10:06:17,198] {subprocess.py:82} INFO - Command exited with return code 0
[2024-02-26 10:06:17,247] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=terceira_dag, task_id=tsk3, execution_date=20240226T100558, start_date=20240226T100611, end_date=20240226T100617
[2024-02-26 10:06:17,311] {local_task_job.py:151} INFO - Task exited with return code 0
[2024-02-26 10:06:17,343] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
